notify/notify.go

```diff
@@ -382,8 +388,8 @@ func (pb *PipelineBuilder) New(
 
 	ms := NewGossipSettleStage(peer)
 	is := NewMuteStage(inhibitor)
-	tas := NewTimeActiveStage(intervener)
-	tms := NewTimeMuteStage(intervener)
+	tas := NewTimeActiveStage(intervener, pb.metrics)
+	tms := NewTimeMuteStage(intervener, pb.metrics)
 	ss := NewMuteStage(silencer)
```

Comment:
Have we considered adding silenced and inhibited notifications as part of this metric?  I.e. we could also pass the metrics into `NewMuteStage` here for silences and on line 390 for inhibitions, and increment based on `muted` in that stage as well.

I ask because we didn't specifically tie the metric to mutes (it's not called `num_notifications_muted_total`) and then users could track suppressions across all different methods.

Reply:
I'm not opposed to that :+1: